boards/common/esp32c3/include/arduino_board_common.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C)  2022 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_esp32c3
+ * @{
+ *
+ * @file
+ * @brief       Common board definitions for the Arduino API
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef ARDUINO_BOARD_COMMON_H
+#define ARDUINO_BOARD_COMMON_H
+
+#include "arduino_pinmap.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * @brief   Look-up table for the Arduino's digital pins
+ */
+static const gpio_t arduino_pinmap[] = {
+    ARDUINO_PIN_0,
```

Comment:
Could we move this to a common place and do

```suggestion
#ifdef ARDUINO_PIN_0
    ARDUINO_PIN_0,
#else
    GPIO_UNDEF,
#endif
```

But this is for another PR :wink:

Reply:
Hm, I don`t know. Not all boards use the defines for `arduino_pinmap`. For example, the `sodaq-*` and `esp8266*` boards use the GPIOs directly. Also, the absolute number of Arduino pins differs from board to board.